src/app/helptext/sharing/index.ts

```diff
@@ -6,5 +6,17 @@ export * from "./nfs/nfs";
 export * from "./smb/smb";
 export * from "./webdav/webdav";
 
-export const delete_share_message = T("The sharing configuration will be removed.\
- Data in the share dataset will not be affected.");
+export const shared = {
+    delete_share_message: T("The sharing configuration will be removed.\
+    Data in the share dataset will not be affected."),
+
+    dialog_title: T("Enable service"),
+    dialog_message: T("Enable this service to start automatically."),
```

Comment:
Where is this dialog shown?  In a nightly build, I don't see any snackbar or dialog when clicking the "start automatically" checkbox.  It doesn't need one, the checkbox itself shows whether it is set.

Reply:
It only appears if the service is disabled when you save the form.